tests/test_gpu_examples.py

```diff
@@ -320,7 +320,11 @@ def test_4bit_adalora_causalLM(self):
         """
         model_id = "facebook/opt-350m"
 
-        model = AutoModelForCausalLM.from_pretrained(model_id, load_in_4bit=True)
+        model = AutoModelForCausalLM.from_pretrained(
+            model_id,
+            load_in_4bit=True,
+            device_map={'': 'cuda:0'},  # fix for >3 GPUs
```

Comment:
```suggestion
            device_map={'': PartialState().process_index},  # fix for >3 GPUs
```
After importing `PartialState` from accelerate on top of the file